File: src/org/irislang/jiris/core/exceptions/fatal/IrisConstanceNotFoundException.java
package org.irislang.jiris.core.exceptions.fatal;

import org.irislang.jiris.core.exceptions.IrisRuntimeException;

/**
 * Created by yuwen on 2017/7/6 0006.
 */
public class IrisConstanceNotFoundException extends IrisFatalException {
    public IrisConstanceNotFoundException(String fileName, int lineNumber, String message) {
        super(fileName, lineNumber, message);
    }

    @Override
    public String GetFatalExceptionName() {
        return "ConstanceNotFoundIrregular";
    }
}

File: src/main/java/org/irislang/jiris/core/exceptions/fatal/IrisConstanceNotFoundException.kt
package org.irislang.jiris.core.exceptions.fatal

import org.irislang.jiris.core.exceptions.IrisRuntimeException

/**
 * Created by yuwen on 2017/7/6 0006.
 */
class IrisConstanceNotFoundException(fileName: String, lineNumber: Int, message: String) : IrisFatalException(fileName, lineNumber, message) {

    override fun GetFatalExceptionName(): String {
        return "ConstanceNotFoundIrregular"
    }
}
